src/Tasks/Microsoft.NET.Build.Tasks/IPackageResolver.cs

```diff
@@ -9,5 +10,6 @@ internal interface IPackageResolver
     {
         string GetPackageDirectory(string packageId, NuGetVersion version);
         string GetPackageDirectory(string packageId, NuGetVersion version, out string packageRoot);
+        string ResolvePackageAssetPath(LockFileTargetLibrary package, string relativePath);
```

Comment:
Is this new method actually used?

Reply:
It's used on line 1036 (not part of these changes) but since the type of `packageResolver` changed from `NuGetPackageResolver` to `IPackageResolver` to allow for mocks, the interface needed to learn the method.